### ISSUE TITLE
Could not create score database.

### ISSUE THREAD

**Issue by Artrassin** [fluddokt/opsu#104]
**Version:** 0.15.0a
**OS:** Linux (armv7l)
**JRE:** The Android Project 0
BOARD: unknown
FINGERPRINT: lge/b2lds_global_com/b2lds:5.0.2/LRX22G/152532131cf4c:user/release-keys
HOST: si-rd10-bx37
MODEL: LG-D690
INCREMENTAL: 152532131cf4c
RELEASE: 5.0.2

**Error:** `Could not create score database.`
**Stack trace:**
```
java.sql.SQLException: android.database.sqlite.SQLiteException: no such table: info (code 1): , while compiling: INSERT OR IGNORE INTO info(key, value) VALUES('version', 20170201)
	at java.lang.reflect.Constructor.newInstance(Native Method)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:288)
	at org.sqldroid.SQLDroidConnection.chainException(SQLDroidConnection.java:158)
	at org.sqldroid.SQLiteDatabase.execSQL(SQLiteDatabase.java:170)
	at org.sqldroid.SQLDroidStatement.execute(SQLDroidStatement.java:96)
	at itdelatrisu.opsu.db.ScoreDB.createDatabase(ScoreDB.java:189)
	at itdelatrisu.opsu.db.ScoreDB.init(ScoreDB.java:115)
	at itdelatrisu.opsu.db.DBController.init(DBController.java:50)
	at itdelatrisu.opsu.Opsu.main(Opsu.java:202)
	at fluddokt.opsu.fake.GameOpsu.create(GameOpsu.java:206)
	at com.badlogic.gdx.backends.android.AndroidGraphics.onSurfaceChanged(AndroidGraphics.java:241)
	at android.opengl.GLSurfaceView$GLThread.guardedRun(GLSurfaceView.java:1511)
	at android.opengl.GLSurfaceView$GLThread.run(GLSurfaceView.java:1239)
Caused by: android.database.sqlite.SQLiteException: no such table: info (code 1): , while compiling: INSERT OR IGNORE INTO info(key, value) VALUES('version', 20170201)
	at android.database.sqlite.SQLiteConnection.nativePrepareStatement(Native Method)
	at android.database.sqlite.SQLiteConnection.acquirePreparedStatement(SQLiteConnection.java:897)
	at android.database.sqlite.SQLiteConnection.prepare(SQLiteConnection.java:508)
	at android.database.sqlite.SQLiteSession.prepare(SQLiteSession.java:726)
	at android.database.sqlite.SQLiteProgram.<init>(SQLiteProgram.java:58)
	at android.database.sqlite.SQLiteStatement.<init>(SQLiteStatement.java:31)
	at android.database.sqlite.SQLiteDatabase.executeSql(SQLiteDatabase.java:1784)
	at android.database.sqlite.SQLiteDatabase.execSQL(SQLiteDatabase.java:1715)
	at org.sqldroid.SQLiteDatabase.execSQL(SQLiteDatabase.java:163)
	... 9 more
```

**Comment by zetsyuko** [comment]
Hi, you know we have the same issue 😒

**Comment by fluddokt** [comment]
Should be fixed in 0.15.0b
